cli/integration_tests/run_logging/log_prefix.t

```diff
@@ -0,0 +1,56 @@
+Setup
+  $ . ${TESTDIR}/../setup.sh
+  $ . ${TESTDIR}/setup.sh $(pwd) ./monorepo
+
+
+  $ ${TURBO} run build --log-prefix=none
+  \xe2\x80\xa2 Packages in scope: app-a (esc)
+  \xe2\x80\xa2 Running build in 1 packages (esc)
+  \xe2\x80\xa2 Remote caching disabled (esc)
+  cache miss, executing 74eb1b46ce8b29d3
+
+  > build (esc)
+  > echo 'build app-a' (esc)
+  
+  build app-a
+  
+   Tasks:    1 successful, 1 total
+  Cached:    0 cached, 1 total
+    Time:\s*[\.0-9]+m?s  (re)
+  
+  $ cat app-a/.turbo/turbo-build.log
+
+  > build (esc)
+  > echo 'build app-a' (esc)
```

Comment:
Having trouble validating this output with prysk: https://github.com/Nicoretti/prysk/issues/163